content/en/user-guide/tools/cloud-pods/remotes/index.md

```diff
@@ -0,0 +1,116 @@
+---
+title: "Cloud Pod Remotes"
+weight: 5
+categories: ["LocalStack Pro", "Tools", "Persistence", "CLI"]
+description: >
+  Guide to the usage of Cloud Pods remotes
+aliases:
+  - /tools/cloud-pods/remotes/
+---
+
+{{< alert title="Beta">}}
+Cloud Pod Remotes is a feature that is currently in beta.
+{{< /alert >}}
+
+A remote is the place where your Cloud Pods are stored.
+By default, we store the Cloud Pods' artifacts in the LocalStack platform.
+However, for various reasons, e.g., data regulation and sovereignty, you might want to store Cloud Pods in a different location that is under your control.
+
+We currently offer two alternative remotes:
+- S3 bucket remote storage;
+- [ORAS](https://oras.land/) (OCI Registry as Storage) remote storage.
+
+Our CLI has been expanded and now offers commands to create and delete remotes:
+
+```bash
+localstack pod remote --help
+Usage: localstack pod remote [OPTIONS] COMMAND [ARGS]...
+
+  Manage cloud pod remotes
+
+Options:
+  -h, --help  Show this message and exit.
+
+Commands:
+  add     Add a remote
+  delete  Delete a remote
+```
+
+Moreover, the `localstack pod remotes` command will show all the registered remotes.
+
+## S3 bucket remote storage
+The S3 storage lets you store Cloud Pods into any arbitrary S3 bucket.
+The first step is to export proper AWS credentials in the terminal session.
+
+```bash
+export AWS_ACCESS_KEY_ID=...
+export AWS_SECRET_ACCESS_KEY=...
+```
+
+A possible option is to obtain credentials via [AWS SSO CLI](https://github.com/synfinatic/aws-sso-cli).
+
+Afterwards, we have to add a new remote that explicitly targets a S3 bucket.
+With the command below, we are creating a new remote called `s3-storage-aws` that will store the Cloud Pods' artifacts into a S3 bucket name `ls-pods-bucket-test`.
+The `access_key_id` and `secret_access_key` placeholder will make sure that the AWS credentials are correctly passed to the container.
+
+```bash
```

Comment:
nit: same as above, better to be replaced with a `<command>` (there's plenty of examples in this repo.. 👍 )